internal/automation-scripts/BuildStaticScanZip.sh

```diff
@@ -48,7 +48,21 @@ do
 				find $STATIC_HOME/$product -regex $actual_pattern -exec cp {} $STATIC_HOME/$product-work \;
 			else
 				echo "$SCRIPT_TAG Copying files that match the pattern: $pattern"
-				find $STATIC_HOME/$product -name $pattern -exec cp {} $STATIC_HOME/$product-work \;
+                        	#Find and Copy all the js,jsp and jaggery files to scan artifact
+                        	if [[ $pattern == *".js"* ]] || [[ $pattern == *".jag"* ]] ;then
+                        		for file in $(find $STATIC_HOME/$product -name $pattern)
+                        		do
+                            			#Create same directory structure for each file
+                            			dir=$(dirname "$file")
+                            			base_path=$STATIC_HOME/$product
+                            			suffix_of_dir="${dir//$base_path/}"
```

Comment:
Let's add comments about what is done here. Might be hard to understand later.

Reply:
Resolved.